test/jose-jws-test.html

```diff
@@ -32,18 +33,18 @@
                             b64uxp.test(signature.payload) &&
                             b64uxp.test(signature.signature);
                 });
-            }), true, "got right JSON serialization format");
+            }), true, "got right JSON serialization format 1");
```

Comment:
I think the '1' is a remnant of digging into the problem. Should be removed.

Reply:
agree, should be removed.

Reply:
Fixed :D